src/elm/View/Section/FacialRecognition.elm

```diff
@@ -99,16 +101,68 @@ viewImage delay state fadeImage =
                 )
             , Html.Attributes.style "transition" (infoTransition fadeImage.isBlank delay)
             ]
-            [ Html.p [] [ Html.text "Extracted Information" ]
-            , Html.p [] [ Html.text "Name: █████ ███████" ]
-            , Html.p [] [ Html.text "Job Title: █████ █████████" ]
-            , Html.p [] [ Html.text "City of Residence: ██████" ]
-            , Html.p [] [ Html.text "Place of Work: ██████" ]
-            , Html.p [] [ Html.text "Email: ██████@██████.com" ]
-            ]
+            ([ Html.p [] [ Html.text (t ProfileInfoHeading) ]
+             ]
+                ++ viewProfileInfoList (profileInfoFromSrcId fadeImage.srcId)
+            )
         ]
 
 
+profile1InfoList : List InfoLabel
+profile1InfoList =
+    -- Type of info (Name, Job, City, Work, Email)
```

Comment:
@stuartleech Let me know if you need a hand editing these... I tried to make it as easy as possible.